datadog/agent/params.go

```diff
@@ -94,7 +96,20 @@ func WithTelemetry() func(*Params) error {
     metrics:
       - "*"
 `
-		p.extraAgentConfig = append(p.extraAgentConfig, "telemetry.enabled: true")
+		p.extraAgentConfig = append(p.extraAgentConfig, pulumi.String("telemetry.enabled: true"))
 		return WithIntegration("openmetrics.d", config)(p)
 	}
 }
+
+func WithFakeintake(fakeintake fakeintake.PulumiData) func(*Params) error {
+	return func(p *Params) error {
+		// configure metrics and check run intake
+		extraConfig := pulumi.Sprintf(`dd_url: http://%s:80
+logs_config.logs_dd_url: %s:80
+logs_config.logs_no_ssl: true
+logs_config.force_use_http: true
+logs_enabled: true`, fakeintake.Url, fakeintake.Url)
```

Comment:
`logs_enabled: true` we should let the user decider to enable it or not? What do you think?